### DISCUSSION TITLE
Explore AI Prompting and Technical Content Creation with our latest Experience

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#102036] [GitHub Education]
### Hi everyone,

<img width="477" alt="Screenshot 2024-01-18 at 3 31 53 PM" src="https://github.com/community/community/assets/107881423/1b6a7640-fdd6-47a7-ac81-71906ee69119">

This [Experience](https://education.github.com/experiences/ai_prompt_tech_writing) gathers together the tools, top tips and resources you'll need to get started becoming a technical wizard with words! 

Technical writing is mostly employed in technical and operational fields and is therefore essential to a variety of enterprises and industries as a result. Without it, it would be difficult or impossible to comprehend or carry out certain operations, complex subjects and understand specific details of new products and tools. 

Prompt engineering refers to specifically designing a prompt in such a way that you receive better results from the AI. With Copilot Chat now available widely, now’s the time to brush up on your prompting skills to get the most out of these powerful helpers.

**Student Developer Pack partners and products featured in this Experience include:** 

- Copilot & Copilot Chat
- Microsoft Word
- Bump.sh (API docs, changelogs and more)
- Educative (Prompt Engineering Essentials course)
- POEditor
- Markdown & the Communicating with Markdown Experience
- Visme - for image editing
- Microsoft Learn - Technical Documentation: for in-depth articles on Microsoft developer tools and technologies.

**Additional resources**

- From GitHub DevRel: [Prompt Engineering for AI](https://dev.to/github/prompt-engineering-for-ai-what-is-prompt-engineering-and-how-to-get-good-results-from-ai-engines-5ch6)
- From OpenAI - [The Art of AI Prompt Crafting: A Comprehensive Guide for Enthusiasts](https://community.openai.com/t/the-art-of-ai-prompt-crafting-a-comprehensive-guide-for-enthusiasts/495144)
- Hear from a Technical Writer: [What I Wish I Knew about being a Technical Writer](https://learn.microsoft.com/en-us/shows/what-i-wish-i-knew/what-i-wish-i-knew-about-being-a-technical-writer)
- From GitHub: [GitHub Prompting Copilot Chat & Accessibility](https://github.blog/2023-10-09-prompting-github-copilot-chat-to-become-your-personal-ai-assistant-for-accessibility/)
- DAIR’s Prompt Engineering Guide https://www.promptingguide.ai/ 
- Copilot Can Do That?: [10 unexpected ways to use GitHub Copilot](https://github.blog/2024-01-22-10-unexpected-ways-to-use-github-copilot/)

**Open Source Recommendations**

- [LanguageTool](https://languagetool.org/) - Open-source writing assistant.
- [Thought Source](https://github.com/OpenBioLink/ThoughtSource)
- [Notepad++](https://notepad-plus-plus.org/) 

[Get Started Here](https://education.github.com/experiences/ai_prompt_tech_writing)

**Comment by ThaminduSulakshana** [comment]
Importance of Technical Writing: Highlighting the significance of technical writing in various fields for understanding complex subjects and tools.

Prompt Engineering for AI: Discussing the importance of crafting prompts effectively to yield better results from AI, particularly focusing on Copilot Chat.

Student Developer Pack Partners and Products: Listing various tools and resources beneficial for technical writing and prompt engineering, including Copilot & Copilot Chat, Microsoft Word, Bump.sh, Educative, POEditor, Markdown, Visme, and Microsoft Learn.

Additional Resources: Mentioning supplementary materials such as guides, articles, and courses from GitHub, OpenAI, DAIR, and others.

Open Source Recommendations: Providing suggestions for open-source writing tools like LanguageTool, Thought Source, and Notepad++.

Overall, this collection serves as a comprehensive guide for individuals looking to enhance their technical writing skills and maximize the potential of AI assistance in writing tasks.

**Comment by aklagnanda01** [comment]
_
